src/test/getAssetFromKV.ts

```diff
@@ -88,6 +88,45 @@ test('getAssetFromKV non ASCII path support', async t => {
   }
 })
 
+test('getAssetFromKV supports browser percent encoded URLs', async t => {
+  mockGlobal()
+  const event = getEvent(new Request('https://example.com/%not-really-percent-encoded.html'))
+  const res = await getAssetFromKV(event)
+
+  if (res) {
+    t.is(await res.text(), 'browser percent encoded')
+  } else {
+    t.fail('Response was undefined')
+  }
+})
+
+test('getAssetFromKV supports user percent encoded URLs', async t => {
+  mockGlobal()
+  const event = getEvent(new Request('https://blah.com/%2F.html'))
+  const res = await getAssetFromKV(event)
+
+  if (res) {
+    t.is(await res.text(), 'user percent encoded')
+  } else {
+    t.fail('Response was undefined')
+  }
+})
```

Comment:
Those test cases is introduced in #104. They are included in this PR as well to show nothing will be broken after this PR.